src/Admin/SharedBlockAdmin.php

```diff
@@ -38,6 +38,9 @@ protected function generateBaseRouteName(bool $isChildAdmin = false): string
         return sprintf('%s_%s', parent::generateBaseRouteName($isChildAdmin), 'shared');
     }
 
+    /**
+     * @param ProxyQueryInterface<object> $query
+     */
```

Comment:
You don't need it here.
```suggestion
```

Reply:
I'm trying to solve a lint issue in this. file :'(